src/DropDown/DropDown.component.tsx

```diff
@@ -123,8 +126,21 @@ export class DropDown extends React.Component<DropDownProps> {
     > = React.createRef();
     private rootElement: any;
     // Observable Subscriptions
-    private hoveredSub$: Subscription;
-    private clickedSub$: Subscription;
+    private hoveredSub: Subscription;
+    private clickedSub: Subscription;
+    private escapeSub: Subscription;
+    private outsideClickSub: Subscription;
+
+    isOpen(): boolean {
+        const { open, trigger = 'both' } = this.props;
+        const { clicked, hovered } = this.state;
+
+        return open !== undefined
+            ? open
+            : (trigger === 'click' && clicked) ||
```

Comment:
Since there are comparison's to the value held in `trigger` in a couple of places, it might be worth making an enum of possible values so there's no 'magic numbers'.

Reply:
Gonna save this for another date as well